src/Cake.Issues.Markdownlint.Tests/LogFileFormat/MarkdownlintLogfileHelperTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace Cake.Issues.Markdownlint.Tests.LogFileFormat
+{
+    using Cake.Issues.Markdownlint.LogFileFormat;
+    using Cake.Issues.Testing;
+    using Shouldly;
+    using Xunit;
+
+    public sealed class MarkdownlintLogFileHelperTests
+    {
+        public sealed class TheCtor
+        {
+            [Fact]
+            public void GetRegexForMarkdownIssueLine_Should_Return_Correct_Pattern_For_Markdownlint_Cli_Smaller_Than_0_19_0()
+            {
+                // Given / When
+                // Issue line Markdownlint-cli version < 0.19.0
```

Comment:
Comment is not required. Test case name should be sufficient (Same for other test cases)